modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/output/callback/DeleteTableCallback.java

```diff
@@ -38,30 +39,34 @@
     private boolean convertToStreamEvent;
     private StateEventPool stateEventPool;
     private StreamEventPool streamEventPool;
-    private StreamEventConverter streamEventConvertor;
+    private StreamEventConverter streamEventConverter;
 
     public DeleteTableCallback(Table table, CompiledCondition compiledCondition, int matchingStreamIndex,
                                boolean convertToStreamEvent, StateEventPool stateEventPool,
-                               StreamEventPool streamEventPool, StreamEventConverter streamEventConvertor) {
+                               StreamEventPool streamEventPool, StreamEventConverter streamEventConverter,
+                               String queryName) {
         this.matchingStreamIndex = matchingStreamIndex;
```

Comment:
Is it ok to introduce an API change here? If this method is only used in OutputParser.java class, it is ok.

Reply:
Yes, it will only be used at the OutputParser. So it won't break the code :)

Reply:
Thank you for the clarification:+1: